usefulUtilities/sitV2/appResources/appData/sitClientV2.js

```diff
@@ -178,11 +180,12 @@
         for (var i in roles) { // restore roles to prevent overlap
             // -----------------THIS IS PRINTING ERRORS WHEN IN HMD WHEN CHANGING SEATS---------------------
             // Rig::restoreRoleAnimation could not find role ***ROLE NAME***
-            MyAvatar.restoreRoleAnimation(roles[i]); 
+            if(changedSeats){
```

Comment:
```suggestion
            if (changedSeats){
```

Reply:
Ack

Reply:
please fix - looks like you missed this in your latest commit (or didn't do it)